src/gui/guiFormSpecMenu.cpp

```diff
@@ -2506,110 +2795,17 @@ void GUIFormSpecMenu::drawMenu()
 	/*
 		Draw backgrounds
 	*/
-	for (const GUIFormSpecMenu::ImageDrawSpec &spec : m_backgrounds) {
```

Comment:
Wouldn't it be possible to use `m_backgrounds`, but with a custom "container rect" as offset?

Reply:
What would be the benefit of this?

Reply:
It would mean that at least these controls are drawn in the same function, separated from the parsing code. However, not many other elements do it this way so I guess it's OK to keep this change.